pydis_site/apps/api/models/bot/infraction.py

```diff
@@ -69,6 +69,14 @@ class Infraction(ModelReprMixin, models.Model):
         help_text="Whether a DM was sent to the user when infraction was applied."
     )
 
+    jump_url = models.CharField(
+        default='',
```

Comment:
What was the reason behind using an empty string here, rather than making it nullable?

Reply:
Answered here: https://github.com/python-discord/bot/pull/2372#discussion_r1118095635

Reply:
I've now replaced CharField with URLField with `null=True` in https://github.com/python-discord/site/pull/824/commits/ed3577652d620057cc87bbc74775548066cfdc8f.